packages/itwin/tree-widget/src/components/trees/stateless/common/UseHierarchyFiltering.tsx

```diff
@@ -116,10 +118,12 @@ interface MatchingInstancesCountProps {
 }
 
 function MatchingInstancesCount({ filter, defaultHierarchyLevelSizeLimit, hierarchyLevelDetails }: MatchingInstancesCountProps) {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Looks like `useDebouncedAsyncValue` returns `inProgress` flag https://github.com/iTwin/appui/blob/5995ba38b3c053d8d2551a7ef460e33b0e0fc466/ui/components-react/src/components-react/common/UseDebouncedAsyncValue.ts#L80C27-L80C38

Reply:
Read the entire description but somehow missed the flag. Changed.